servant-quickcheck.cabal

```diff
@@ -1,5 +1,5 @@
 name:               servant-quickcheck
-version:            0.0.9.1
+version:            0.1.0.0
```

Comment:
could you please add a changelog entry that justifies why we're bumping major version?

Reply:
Of course! This is a wip :3

Reply:
The reason is that this is a breaking change, mainly due to having to adapt to features added by servant itself

Reply:
Sure, no problems with that, I just need a changelog entry. :)

Reply:
can you check whether the one I made is fine? I also am quite confused on this `yaml` format; can we perhaps convert that to something else? sorry I don't actually know this.